ods_ci/tests/Tests/650__distributed_workloads/test-run-kueue-e2e-tests.robot

```diff
@@ -0,0 +1,99 @@
+*** Settings ***
+Documentation     Kueue E2E tests - https://github.com/opendatahub-io/kueue.git
+Suite Setup       Prepare Kueue E2E Test Suite
+Suite Teardown    Teardown Kueue E2E Test Suite
+Library           OperatingSystem
+Library           Process
+Library           OpenShiftLibrary
+
+
+*** Variables ***
+${KUEUE_DIR}            kueue
+${KUEUE_REPO_URL}       %{KUEUE_REPO_URL=https://github.com/opendatahub-io/kueue.git}
+${KUEUE_REPO_BRANCH}    %{KUEUE_REPO_BRANCH=main}
+${JOB_GO_BIN}           %{WORKSPACE=.}/go-bin
+${KUBECONFIG}           %{WORKSPACE=.}/kconfig
+${WORKER_NODE}          ${EMPTY}
+
+
+*** Test Cases ***
+Run E2E test
+    [Documentation]    Run ginkgo E2E single cluster test
+    [Tags]  Kueue
```

Comment:
Do you plan to add also tag for tier (like Sanity, Tier2, etc.)?

Reply:
I am thinking to add these tags later when kueue will be integrated with Distributed Workloads